content/en-us/designs/protocol-translation/_index.md

```diff
@@ -1,49 +1,79 @@
 ---
-title: "Industrial Data Extraction"
+title: "Protocol Translation"
 weight: 50
 ---
 
-{{< synopsis-iiot-extraction >}}
+{{< synopsis-protocol-translation >}}
 <!--more-->
 
 ## Challenge
+At the edge of an IoT solution, devices are often interconnected and have to exchange data locally, i.e., without moving data over the internet.
+As some of these devices may use different protocols that are not compatible with each other, protocols have to be translated.
+For instance in industrial IoT solutions, the interfaces of machines often use different protocols and extracting the information from these machines for monitoring requires their translations into a common format that is usable by the backend infrastructure or by a local actor.
+Other examples are smart home deployments, where different devices use different standards, but still need to exchange data, e.g., to control PV power generation depending on the internal power consumption of the home.
+
+Typically, moving all data in an untranslated form over the internet into a backend that finally does the translation is not feasible for several reasons:
+
+* The volume or velocity of the data generated by the local devices is too high for constant streaming of data generated by the devices in their respective formats.
+* The devices use a communication stack that is not based on the [IP-Protocol]({{< ref "https://en.wikipedia.org/wiki/Internet_Protocol" >}}), e.g., serial connections.
+* The devices use a technology for communication that is not usable for communication over the internet, e.g., [Bluetooth]({{< https://en.wikipedia.org/wiki/Bluetooth >}}).
+* Connecting the devices directly to the internet or communication using such a prototcol would create a security risk, e.g., because the protocol does not support encryption or authentication mechanisms.
+
+The protocol translation design described below will solve these challenge.
+
+
+
+## Solution
+IoT solutions address these challenges by applying the [Gateway]({{< ref "/designs/gateway" >}}) in combination with edge logic running on the gateway.
+The edge logic on the gateway executes the translation of the industrial protocols into a common data format/protocol that can be processed by backend IT systems such as databases, data lakes or webservices.
+An example of an architecture for such a solution may be as follows:
+
+![IIoT Extraction Design](ProtocolTranslation.png)
```

Comment:
This link should change from "IIoT Extraction Design" to "Protocol Translation Design"